tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Constraints.cpp

```diff
@@ -287,6 +287,22 @@ void test_two_index_constraint_analytic(
   const auto& d_gauge_function =
       get<Tags::deriv<GaugeH, tmpl::size_t<3>, Frame::Inertial>>(gh_derivs);
 
+  // Populate spacetime derivative of gauge function
+  const auto d4_gauge_function = [&x, &d_gauge_function]() {
+    auto tmp =
+        make_with_value<tnsr::ab<DataVector, 3, Frame::Inertial>>(x, 0.0);
+    for (size_t a = 0; a < 4; ++a) {
+      for (size_t i = 0; i < 3; ++i) {
+        tmp.get(i + 1, a) = d_gauge_function.get(i, a);
+      }
+      // Populate the time derivative with sequential integers to serve
+      // as a check that those are not being used in calculating
+      // constraints
+      tmp.get(0, a) = 11.0 + a;
```

Comment:
If you don't want to use them, wouldn't it be better to fill them with signaling nans?

Reply:
sure, changed.